Tests/Util/ApiUtil.cs

```diff
@@ -19,7 +19,7 @@ static ApiUtil()
         {
             _configuration = new Configuration()
             {
-                Timeout = 30000,
+                Timeout = 7500,
```

Comment:
why so low timeout?

Reply:
There are quite a few retries with sleep in between, it made tests take too long to complete